convert/src/label_studio_to_badgerdoc/label_studio_to_badgerdoc_use_case.py

```diff
@@ -394,15 +432,20 @@ def request_annotations_to_post_categories(
         ]
 
     def request_annotation_to_post_annotations(
-        self, annotation_job_id_created: int, file_id_in_assets: int, document_labels: Set[str]
+        self,
+        annotation_job_id_created: int,
+        file_id_in_assets: int,
+        document_labels: Set[str],
     ) -> None:
         annotations_post_url = f"{settings.annotation_service_url}annotation/{annotation_job_id_created}/{file_id_in_assets}"
 
         page = self.badgerdoc_format.badgerdoc_annotation
         annotations_post_body = {
             "base_revision": None,
             "pipeline": 0,
-            "pages": [page.dict(),],
```

Comment:
I believe linter have made formatting because of the redundant comma

Reply:
you're right. I've deleted this comma and undid this change